addons/common/fnc_addLoadoutFilter.sqf

```diff
@@ -0,0 +1,32 @@
+#include "script_component.hpp"
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_addLoadoutFilter
+Description:
+    Add a filter for CBA_fnc_getUnitLoadout.
+Parameters:
+    _function - The function you wish to execute. <CODE>
+Passed Arguments:
+    _this <ARRAY>
+        0: _loadout - A getUnitLoadout array
+Returns:
+    Nothing.
+Examples:
+    Remove the radio from loadouts
+    (begin example)
+        [{
+            params ["_loadout"];
+            if ((_loadout select 9) select 2 == "ItemRadio") then {
+                (_loadout select 9) set [2, ""];
+            };
+            _loadout
+        }] call CBA_fnc_addLoadoutFilter;
+    (end)
+Author:
+    SynixeBrett
+---------------------------------------------------------------------------- */
+
+params [["_function", {}, [{}]]];
+
+if (_function isEqualTo {}) exitWith {-1};
```

Comment:
Remove `-1` and add `nil` at the end of file since header says function returns nothing.

Reply:
It could return false, and true if it worked.